.github/workflows/dotnet-core-desktop.yml

```diff
@@ -85,4 +84,34 @@ jobs:
       run: msbuild $env:Solution_Name /t:Restore /p:Configuration=$env:Configuration
       env:
         Configuration: ${{ matrix.configuration }}
+        
+        
+  build_ubuntu:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+
+    env:
+      Solution_Name: eawx-build.sln                         # Replace with your solution name, i.e. MyWpfApp.sln.
+      Test_Project_Path: eawx-build-test/eawx-build-test.csproj                 # Replace with the path to your test project, i.e. MyWpfApp.Tests\MyWpfApp.Tests.csproj.
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      
+    steps:
+      - name: Install zlib
+        run: sudo apt-get install -y zlib1g
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.7.2
```

Comment:
Is there a need for the specific version, otherwise you could just use `actions/setup-dotnet@v1`